package.json

```diff
@@ -84,6 +84,7 @@
     "webpack": "^5.72.0"
   },
   "resolutions": {
-    "axe-core": "^4.3.5"
+    "axe-core": "^4.3.5",
+    "prettier": "^2.6.2"
```

Comment:
I've added this resolution because changes in Storybook packages (many depend on `>=2.2.1 <=2.3.0`) somehow made prettier switch to the `2.3.0` executable (according to `yarn prettier --version`).

This ensures we're using the version from foundry (2.6.2). One downside is that we'd need to unpin/update it when we bump Foundry.

It flagged a couple of linting errors that I've fixed in a separate commit.